Dockerfile

```diff
@@ -37,4 +37,5 @@ ENV ADDRESS="" \
     SETUP="false" \
     AUTO_UPDATE="true" \
     LOG_LEVEL="" \
-    BINARY_DIR="/app/config/bin"
+    BINARY_DIR="/app/bin" \
```

Comment:
nit: I'm thinking we don't even need this env anymore since we are somehow going back to the default. We only need the BINARY_STORE. In that case we wouldn't have to worry so much about changing the /app/bin in every file

Reply:
Added a commit to fix that